.readthedocs.yml

```diff
@@ -1,15 +1,14 @@
-
 version: 2
```

Comment:
These changes are necessary as the read-the-docs site itself has changed its api.
See[ this build failure](https://readthedocs.org/projects/plutus-apps/builds/22430196/) for example